spark-jobs/src/main/scala/filodb/downsampler/chunk/BatchExporter.scala

```diff
@@ -218,10 +209,13 @@ case class BatchExporter(downsamplerSettings: DownsamplerSettings, userStartTime
   }
 
   private def makeLabelString(labels: collection.Map[String, String]): String = {
-    val inner = labels
-      .map {case (k, v) => (k, getExportLabelValueString(v))}
-      .map {case (k, v) => s"'$k':'$v'"}
-      .mkString (",")
+    val inner = labels.map {case (k, v) =>
+      if (v.contains (",") ) {
+        String.format ("'%s\':\"%s\"", k, v)
```

Comment:
why the first ' is not escaped?
should it be?
"\'%s\':\"%s\""

Reply:
Good catch-- I'm not sure (the escape may not be needed for the one single-quote). But I will leave this as-is for now since we know it works (we've previously merged this exact diff, but the changes were lost after a merge from `integration` to `main`).

Reply:
or should it be String.format ("'%s':\"%s\"", k, v) ?

Reply:
You may be correct, but I won't touch it for now since this exact diff has worked well in the past.